### PR TITLE
Added new planner for account

### PR DESCRIPTION
**Pull request by dka-odoo**
Task :-  https://www.odoo.com/web#id=10007&view_type=form&model=project.task&action=327

Pad :- https://pad.odoo.com/p/r.GqvrphJqx2Js0AhT

### CODE REVIEW

File: addons/account/views/web_planner_data.xml [hunk 1]
```diff
@@ -0,0 +1,644 @@
...
+                        <li><strong>Clean customer invoices:</strong> easy to create, beautiful and fully featured invoices (payment terms, multiple taxes, discounts, etc).</li>
+                        <li><strong>Automated documents sending:</strong> automatically print, send and deliver your invoices &amp; letters by regular mail or email.</li>
+                        <li><strong>On-the-fly payment reconciliation:</strong> automatic suggestion of outstanding payments when creating an invoice.</li>
+                        <li><strong>Bank interface:</strong> your bank statements are automatically synchronized with your bank.</li>
+                    </ul>
+                </p>
+                <p>If you'd like some help on any subject dont hesitate to contact our Accounting experts by using the <span class="fa fa-question-circle"/> icon on the top bar.</p>
+                <img src="/web_planner/static/src/img/fabien_pinckaers.jpg" class="pull-left mb32"/>
+                <div class="pull-left" style="margin-left: 10px">
```
Comment (hmo-odoo) [inline addons/account/views/web_planner_data.xml]: can you replace existing margin class instead of putting inline css ?

Comment (dka-odoo) [inline addons/account/views/web_planner_data.xml]: done ..!!!

File: addons/account/views/web_planner_data.xml [hunk 2]
```diff
@@ -0,0 +1,644 @@
...
+                    <p class="text-muted">
+                        The multi-currency option allows you to send or receive documents (excluding expense claims) in currencies other than your own, to set up bank accounts in other currencies and run reports on your foreign currency activities.<br/>Note: it's important to configure multi-currency before balancing your Accounts.
+                    </p>
+                    <p>To use the <strong>multi-currency option:</strong>
+                        <ol>
+                            <li> Activate the corresponding option in the <a t-att-href="prepare_backend_url('account.action_account_config')">Accounting Settings</a></li>
+                            <li>Review the list of available currencies (from the Accounting Configuration menu)</li>
+                        </ol>
+                    </p>
+                    <!-- <div class="alert alert-info info_icon" role="alert">
```
Comment (hmo-odoo) [inline addons/account/views/web_planner_data.xml]: why stuff in comment ?

Comment (dka-odoo) [inline addons/account/views/web_planner_data.xml]: because online rates not implemented yet...  check the line 18 in pad...:)

**Pull request closed without merging**